storage/src/adb/any.rs

```diff
@@ -2,20 +2,11 @@
 //! with a key. Its implementation is based on an [Mmr] over a log of state-change operations backed
 //! by a [Journal].
 //!
-//! # Terminology
-//!
-//! A _key_ in the db either has a _value_ or it doesn't. Two types of _operations_ can be applied
-//! to the db to modify the state of a specific key. A key that has a value can change to one
-//! without a value through the _delete_ operation. The _update_ operation gives a key a specific
-//! value whether it previously had no value or had a different value.
-//!
-//! Keys with values are called _active_, and an operation is called _active_ if (1) its key is
-//! active, (2) it is an update operation, and (3) it is the most recent operation for that key.
-//!
```

Comment:
moved this to the mod.rs file since this terminology will apply to other ADB types.

Reply:
For now, I don't see any issue with this (as our `adb`'s are very related) but we should keep an eye on it I think 🤷 .